listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -150,6 +157,8 @@ export default class UserFeedPage extends React.Component<
           review.entity_type
         )}`;
       }
+      case EventType.PERSONAL_RECORDING_RECOMMENDATION:
+        return "personally recommended a track";
```

Comment:
~~In the same discussion about phrasing, how about "UserX recommended you a track" or "UserX sent you a track" ?~~
Now that I saw the other end of it (i.e. on my own feed), I see that wouldn't work ("You recommended you a track").
Back to the drawing board for that one…